### PR TITLE
Add and open by default Fairphone HQ marker. Close #12

### LINKED ISSUE #12: Make marker pop-ups self-explaining
At the moment, I fear that people don't know that every marker has a popup, if they don't accidentally click on one. How can we make marker pop-ups self-explaining?

### PR DESCRIPTION
**Pull request by Roboe**
-  Add and open by default Fairphone HQ marker
-  Center map on Fairphone HQ

Preview: https://roboe.github.io/fprsmap/

I don't like the raw HTML in the code, but... don't know other way to achieve the same functionality.

Also, we'd want to style the popup content, maybe.

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -86,12 +86,32 @@
       maxZoom: 18,
     });
 
+    var fairphoneHqLocation = [52.3771591, 4.9206593];
+
+    var fairphoneHqMarker = L.marker(
+        fairphoneHqLocation,
+        { icon: MARKERICONS.red, riseOnHover: true }
```
Comment (StefanBrand) [inline js/main.js]: Can be left out as generally markers aren't on top of each other because of the cluster.

**Comment by StefanBrand** [comment]
> I don't like the raw HTML in the code, but... 

Me neither it looks awful! 😖 

> don't know other way to achieve the same functionality.

Well, we could `fetchJSON` another file stored locally. 

> Also, we'd want to style the popup content, maybe.

Maybe this helps: http://stackoverflow.com/questions/12606141/how-would-i-customise-the-look-and-feel-of-the-leaflet-popup

Another thing: Can we add the marker to the cluster, so it clusters with the other markers?

**Comment by Roboe** [comment]
> Well, we could `fetch` another file stored locally. 
We can write a template in the HTML file and apply it by JS! That'd be the better, :)

> Another thing: Can we add the marker to the cluster, so it clusters with the other markers?
Yes! I forgot about the cluster, sorry.

**Comment by Roboe** [comment]
If we add the HQ marker to the cluster, it gets grouped and it's not open by default, so it jeopardizes the purpose of this PR, :/

**Comment by StefanBrand** [comment]
Hm, maybe we should drop it then.

I'd give the cluster more priority than the open-by-default popup. If 
the marker is not clustered, it just looks awful.

Am 30.09.2016 um 02:18 schrieb Roberto M.F.:

**Comment by Roboe** [comment]
Ok. Should we leave the Fairphone HQ marker anyway?

**Comment by StefanBrand** [comment]
I'd close it and **not** merge the PR. 

As you said in #12: 
>I think the behaviour is quite known for the mayority of the users
So it will probably not make usability worse.

**Pull request closed without merging**